app/services/foreman_webhooks/webhook_service.rb

```diff
@@ -10,7 +10,13 @@ def initialize(webhook:, event_name:, payload:, headers:, url:)
       @webhook = webhook
       @event_name = event_name
       @payload = payload
-      @rendered_headers = headers
+      # rubocop:disable Lint/RedundantSafeNavigation
+      @rendered_headers = if headers&.is_a? Hash
```

Comment:
This doesn't make sense (also doesn't work, suppresses the headers in the request), since we pass here either `nil` or a `String`: https://github.com/theforeman/foreman_webhooks/blob/3350ac46fbeec4b19f670797d6d6f051834b5698/app/jobs/foreman_webhooks/deliver_webhook_job.rb#L9-L11 from https://github.com/theforeman/foreman_webhooks/blob/3350ac46fbeec4b19f670797d6d6f051834b5698/app/models/webhook.rb#L69-L74 from https://github.com/theforeman/foreman_webhooks/blob/3350ac46fbeec4b19f670797d6d6f051834b5698/app/models/webhook.rb#L121-L126

The best thing I could make is: https://github.com/ofedoren/foreman_webhooks/commit/abc9d7d071e0e0d1c7c986c3e58371f950be4a3b